isis/src/docsys/documents/InstallGuide/InstallGuide.xml

```diff
@@ -59,43 +118,42 @@ one time) may be useful depending on the complexity of your processing requireme
+    <li><a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Building-ISIS3-with-cmake#anaconda-and-isis3-dependencies">Installing Anaconda and satisfying ISIS3 Dependencies</a></li>
+    <li><a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Building-ISIS3-with-cmake#building-isis3">Building ISIS3 With cmake</a></li>
+    <li><a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Building-ISIS3-with-cmake#new-environmental-variable-meanings">New ISIS3 environmental variables and their meanings</a></li>
     <li><a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Building-ISIS3-with-cmake#custom-data-and-test-data-directories">Custom data and test directories</a></li>
     <li><a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Building-ISIS3-with-cmake#cleaning-builds">Cleaning builds</a></li>
     <li><a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Building-ISIS3-with-cmake#building-individual-isis3-applicationsobjects">Building individual applications/objects</a></li>
-    <li><a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Building-ISIS3-with-cmake#building-isis3-documentation">Building ISIS 3 documentation</a></li>
+    <li><a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Building-ISIS3-with-cmake#building-isis3-documentation">Building ISIS3 documentation</a></li>
     <li><a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Building-ISIS3-with-cmake#problems">What to do if you encounter any problems</a></li>
   </ul>
 </p>
 
 <h3>Links for Developers</h3>
 
   <ul>
-    <li><a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Developer-Checklists">ISIS 3 Developer Checklist</a></li>
-    <li><a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Jenkins-CI">Jenkins Continuous Integration Testing</a></li>
-    <li><a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Managing-Custom-ISIS-Dependencies-(WIP)">Managing Custom ISIS 3 Dependencies</a></li>
+    <li><a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Developer-Checklists">ISIS3 Developer Checklist</a></li>
+    <li><a href="https://github.com/USGS-Astrogeology/ISIS3/wiki/Managing-Custom-ISIS-Dependencies-(WIP)">Managing Custom ISIS3 Dependencies</a></li>
   </ul>
 
-<h3>Full ISIS 3 Data Download</h3>
+<h3>Full ISIS3 Data Download</h3>
 
 <p>
     If you have the disk space and network speed, you may want to download all of the ISIS data areas which
     includes all missions supported by ISIS.   This takes over 130 GB of disk space!  If you only have a 10 Mbps
     network connection it will take nearly two days to download.  If you want to acquire only certain mission data
-     <a href="#MissionSpecific">click here</a>.   To download all ISIS 3 data files continue reading.
+     <a href="#MissionSpecific">click here</a>.   To download all ISIS3 data files continue reading.
```

Comment:
The above links are git oriented and I assume conda. This is a bit, `rsync` who now? Maybe we should alert users that data is shipped in a different way. So, I am being a total pain - complaining about narrative above and then talking about jarring here.